<style>
	.accordion-holder {
		max-width: 1000px;
		margin: 20px auto;
	}

	.accordion.active~.panel {
		background: white !important;
	}
</style>

<div class="accordion-holder">
	<button class="accordion">Protocol 1 : NucleoSpin® Plasmid EasyPure protocol mini (Marchery Nagel)</button>
	<div class="panel">
		<h2>1 Cultivate and harvest bacterial cells</h2>
		Use 2–10 mL of a saturated E. coli culture (see page 8, table 3), pellet cells in a standard benchtop microcentrifuge
		for 30 s at > 12,000 x g.
		Discard the supernatant and remove as much of the liquid as possible.
		&gt; 12,000 x g, 30 s
		<h2>2 Cell lysis</h2>
		Add 150 μL Buffer A1. Resuspend the cell pellet completely by vortexing or pipetting up and down. Make sure no cell
		clumps remain before addition of Buffer A2!
		Attention: Check Buffer A2 for precipitated SDS. If a white precipitate is visible, warm the buffer for several
		minutes at 30–40 °C until precipitate is dissolved completely. Cool buffer down to room temperature (18–25 °C) before
		use.
		Add 250 μL Buffer A2. Mix gently by inverting the tube 5 times. Do not vortex to avoid shearing of genomic DNA.
		Incubate at room temperature (18–25 °C) for up to 2 min or until lysate appears clear.
		Add 350 μL Buffer A3. Mix thoroughly by inverting the tube until LyseControl has turned colorless throughout the
		lysate without any traces of blue color. Do not vortex to avoid shearing of genomic DNA!
		+ 150 μL A1
		Resuspend
		+ 250 μL A2
		Mix
		RT, 2 min
		+ 350 μL A3
		Mix
		<h2>3 Clarification of lysate</h2>
		Centrifuge for 3 min at full speed (&gt; 12,000 x g).
		&gt; 12,000 x g, 3 min

		<h2>4 Bind DNA</h2>
		Place a NucleoSpin® Plasmid EasyPure Column into a Collection Tube (2 mL) and decant the supernatant from step 3 onto
		the column.
		Centrifuge for 30 s at 1,000–2,000 x g.
		Discard flow-through and place the spin column back into the collection tube.
		Load supernatant
		1,000– 2,000 x g, 30 s
		<h2>5 Wash and dry silica membrane</h2>
		Add 450 μL Buffer AQ (supplemented with ethanol, see section 3).
		Centrifuge for 1 min at full speed (&gt; 12,000 x g).
		Very carefully discard the collection tube and the flow- through and make sure the spin cup outlet does not touch the
		wash buffer surface. Otherwise repeat the centrifugation step.
		Note: To reduce ethanol carry-over to a minimum for better performance in downstream applications, incubate spin cup
		for 10–15 min at 37 °C to dry silica membrane completely.
		+ 450 μL AQ
		&gt; 12,000 x g, 1 min
		<h2>6 Elute DNA</h2>
		Place the NucleoSpin® Plasmid EasyPure Column in a 1.5 mL microcentrifuge tube (not provided) and add 50 μL Buffer
		AE.
		Incubate for 1 min at room temperature (18–25 °C).
		Centrifuge for 1 min at full speed (&gt; 12,000 x g).
		Note: For more efficient elution procedures and alternative elution buffer (e.g., TE buffer or water) see section
		2.4.
		+ 50 μL AE
		RT, 1 min
		&gt; 12,000 x g, 1 min

	</div>
	<button class="accordion">Protocol 2 : NucleoBond® Xtra plasmid purification midi/maxi (Marchery Nagel)</button>
	<div class="panel">
		<h2>1 Preparation of starter culture</h2>
		Inoculate a 3–5 mL starter culture of LB medium with a single colony picked from a freshly streaked agar plate. Make
		sure that plate and liquid culture contain the appropriate selective antibiotic to guarantee plasmid propagation (see
		section 4.3 for more information). Shake at 37 °C and ~ 300 rpm for ~ 8 h.
		<h2>2 Preparation of large overnight culture</h2>
		Inoculate an overnight culture by diluting the starter culture 1 / 1000 into the given volumes of LB medium also
		containing the appropriate selective antibiotic. Grow the culture overnight at 37 °C and ~ 300 rpm for 12–16 h.
		100 mL 300 mL
		<h2>3 Harvest of bacterial cells</h2>
		Measure the cell culture OD 600 and determine the recommended culture volume.
		V [mL] = 400 / OD 600
		V [mL] = 1200 / OD
		600
		Pellet the cells by centrifugation at 4,500–6,000 x g for ≥ 10 min at 4 °C and discard the supernatant completely.
		Note: Optimal lysis conditions are achieved by a unique ratio of lysis buffer RES, LYS, and NEU to the cell mass. See
		section 4.5 for recommendations concerning the optimal pelleted culture volume for cells containing high-copy
		plasmids and section 4.6 for recommendations concerning cells with low-copy plasmids. Read section 4.5 carefully as
		excess cell input will result in reduced yield. The recommended culture volumes below are calculated for a final OD
		600 of around 4.

		<h2>4 Resuspension (Buffer RES)</h2>
		Resuspend the cell pellet completely in Resuspension Buffer RES + RNase A by pipetting the cells up and down or
		vortexing the cells.
		For an efficient cell lysis it is important that no clumps remain in the suspension.
		Note: Increase RES buffer volume proportionally if more than the recommended cell mass is used (see section 4.8 for
		information on optimal cell lysis and section 4.9 regarding difficult-to-lyse strains).

		<h2>5</h2>
		Cell lysis (Buffer LYS) !
		Check Lysis Buffer LYS for precipitated SDS prior to use. If a white precipitate is visible, warm the buffer for
		several minutes at 30–40 °C until precipitate is dissolved completely. Cool buffer down to room temperature (18–25
		°C).
		Add Lysis Buffer LYS to the suspension.
		Mix gently by inverting the tube 5 times. Do not vortex as this will shear and release contaminating chromosomal DNA
		from cellular debris into the suspension.
		Incubate the mixture at room temperature (18–25 °C) for 5 min.
		Warning: Prolonged exposure to alkaline conditions can irreversibly denature and degrade plasmid DNA and liberate
		contaminating chromosomal DNA into the lysate.
		Note: Increase LYS buffer volume proportionally if more than the recommended cell mass is used (see section 4.8 for
		information on optimal cell lysis).
		8 mL 12 mL
		<h2>6 Equilibration (Buffer EQU)</h2>
		Equilibrate a NucleoBond® Xtra Column together with the inserted column filter with Equilibration Buffer EQU.
		Apply the buffer onto the rim of the column filter as shown in the picture and make sure to wet the entire filter.
		Allow the column to empty by gravity flow. The column does not run dry.
		12 mL 25 mL

		<h2>7</h2>
		Neutralization (Buffer NEU) !
		Add Neutralization Buffer NEU to the suspension and immediately mix the lysate gently by inverting the tube until
		blue samples turns colorless completely! Do not vortex.
		The flask or tube used for this step should not be filled more than two thirds to allow homogeneous mixing. Make sure
		to neutralize completely to precipitate all the protein and chromosomal DNA. The lysate should turn from a slimy,
		viscous consistency to a low viscosity, homogeneous suspension of an off-white flocculate. In addition, LyseControl
		should turn completely colorless without any traces of blue.
		Immediately proceed with step 8. An incubation of the lysate is not necessary.
		Note: Increase NEU buffer volume proportionally if more than the recommended cell mass is used (see section 4.8 for
		information on optimal cell lysis).
		8 mL 12 mL
		<h2>8</h2>
		Clarification and loading !
		Make sure to have a homogeneous suspension of the precipitate by inverting the tube 3 times directly before applying
		the lysate to the equilibrated NucleoBond® Xtra Column Filter to avoid clogging of the filter.
		The lysate is simultaneously cleared and loaded onto the column. Refill the filter if more lysate has to be loaded
		than the filter is able to hold. Allow the column to empty by gravity flow.
		Alternative: The precipitate can be removed by centrifugation at ≥ 5,000 x g for at least 10 min, for example if more
		than double the recommended cell mass was used. If the supernatant still contains suspended matter transfer it to a
		new tube and repeat the centrifugation, preferably at higher speed, or apply the lysate to the equilibrated
		NucleoBond® Xtra Column Filter.
		This clarification step is extremely important since residual precipitate may clog the NucleoBond® Xtra Column. To
		load the column you can either apply the cleared lysate to the equilibrated filter or remove the unused filter
		beforehand. Allow the column to empty by gravity flow.
		Note: You may want to save all or part of the flow-through for analysis (see section 8.1).

		Midi Maxi
		<h2>9</h2>
		1st Wash: Column filter and column !
		(Buffer EQU)
		Wash the NucleoBond® Xtra Column Filter and Nu-cleoBond® Xtra Column with Equilibration Buffer EQU.
		Apply the buffer to the funnel shaped rim of the filter and make sure it is washing out the lysate which is remaining
		in the filter. Omitting this step or just pouring the buffer directly inside the funnel may reduce plasmid yield.
		5 mL 15 mL
		<h2>10 Filter removal</h2>
		Either pull out the NucleoBond® Xtra Column Filter or discard it by turning the column upside down.
		<h2>11</h2>
		2nd Wash: Column only (Buffer WASH) !
		Wash the NucleoBond® Xtra Column with Wash Buffer WASH. It is important to remove the column filter before applying
		the washing buffer to avoid low purity.
		8 mL 25 mL
		<h2>12 Elution (Buffer ELU)</h2>
		Elute the plasmid DNA with Elution Buffer ELU. Collect the eluate in a 15 mL or 50 mL centrifuge tube (not provided).
		Note: Preheating Buffer ELU to 50 °C prior to elution may improve yields for large constructs such as BACs.
		Optional: Determine plasmid yield by UV spectrophotometry in order to adjust desired concentration of DNA in step 15
		and calculate the recovery after precipitation.
		5 mL 15 mL !
		NucleoBond® Xtra Midi / Maxi Plus:
		Proceed with step 13 for the centrifugation protocol after isopropanol precipitation or continue with section 7.3 for
		plasmid concentration and desalting by using the NucleoBond® Finalizer (NucleoBond® Xtra Midi Plus) or NucleoBond®
		Finalizer Large (NucleoBond® Xtra Maxi Plus).

		<h2>13 Precipitation</h2>
		Add room-temperature isopropanol to precipitate the eluted plasmid DNA.
		Vortex thoroughly!
		Centrifuge at ≥ 4,500 x g for ≥ 15 min at ≤ room temperature, preferably at 15,000 x g for 30 min at 4 °C. Carefully
		discard the supernatant.
		3.5 mL 10.5 mL
		<h2>14 Washing and drying</h2>
		Add room-temperature 70 % ethanol to the pellet.
		2 mL 14 mL
		Centrifuge at ≥ 4,500 x g, preferably ≥ 15,000 x g for 5 min at room temperature (18–25 °C).
		Carefully remove ethanol completely from the tube with a pipette tip. Allow the pellet to dry at room temperature
		(18–25 °C).
		Note: Plasmid DNA might be harder to dissolve when over-dried.
		10–15 min 15–30 min
		<h2>15 Reconstitution</h2>
		Dissolve the DNA pellet in an appropriate volume of buffer TE or sterile H2O.
		Depending on the type of centrifugation tube, dissolve under gentle pipetting up and down or constant spinning in a
		sufficient amount of buffer for 10–60 min (3D-shaker).
		Determine plasmid yield by UV spectrophotometry. Confirm plasmid integrity by agarose gel electrophoresis.

	</div>
	<button class="accordion">Protocol 3 : 1% agarose gel</button>
	<div class="panel">
		<ol>
			<li>Measure 1 gram of agarose.</li>
			<li>Add 100 mL of 1x TAE buffer.</li>
			<li>Microwave for 1-3 min until the agarose is completely dissolved.</li>
			<li>Let the agarose cool down to around 50 degrees celcius.</li>
			<li>Add 100 µL GelRed, swirl the flask, and pour slowly into a geltray with well comb.</li>
			<li>Let the gel solidify for 20 to 30 minutes at room temperature</li>
			<li>Load DNA samples mixed with loading buffer / loading dye</li>
			<li>Run at 80 to 150 V for 1 to 2 hours (dependent on gel percentage and voltage used)</li>
		</ol>
	</div>
	<button class="accordion">Protocol 4 : NEB10-béta transformation protocol (New England Biolabs)</button>
	<div class="panel">
		<ol>
			<li>Thaw a tube of NEB 10-beta Competent E. coli cells on ice for 10 minutes. </li>
			<li>Mix gently and carefully pipette 50 µl of cells into a transformation tube on ice.</li>
			<li>Add 1-5 µl containing 1 pg-100 ng of plasmid DNA to the cell mixture. Carefully flick the tube 4-5 times to mix
				cells and DNA. Do not vortex.</li>
			<li>Place the mixture on ice for 30 minutes. Do not mix.</li>
			<li>Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.</li>
			<li>Place on ice for 5 minutes. Do not mix.</li>
			<li>Pipette 950 µl of room temperature NEB 10-beta/Stable Outgrowth Medium into the mixture.</li>
			<li>Place at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.</li>
			<li>Warm selection plates to 37°C.</li>
			<li>Mix the cells thoroughly by flicking the tube and inverting, then perform several 10-fold serial dilutions in
				NEB 10-beta/Stable Outgrowth Medium.</li>
			<li>Spread 50-100 µl of each dilution onto a selection plate and incubate overnight at 37°C. Alternatively, incubate
				at 30°C for 24-36 hours or 25°C for 48 hours.</li>
		</ol>
	</div>
	<button class="accordion">Protocol 5 : Restriction enzyme double digestion</button>
	<div class="panel">
		<ol>
			<li>Per 50 µL reaction mix the following:</li>
			<ul>
				<li>Up to 1 µg</li>
				<li>5 µL 10x Cutsmart Buffer</li>
				<li>10 units of restriction enzyme one</li>
				<li>10 units of restriction enzyme two</li>
				<li>Fill reaction to 50 µL with nuclease free water</li>
			</ul>
			<li>Incubate at 37 degrees Celsius for two hours to overnight</li>
			<li>Inactivate enzymes by incubating for 20 minutes at 65 degrees Celcius</li>
		</ol>
	</div>
	<button class="accordion">Protocol 6 : Chemocompetent cells</button>
	<div class="panel">
		<h2>Materials:</h2>
		<ul>
			<li>Overnight Culture from the right bacteria.</li>
			<li>Tfb1, icecold: 30 mM natriumacetaat, 100mM rubidium chloride,10 mM calcium chloride, 50mM mangaan chloride, 15 %
				glycerol; pH 5,8 achieved by adding diluted NaOH. Sterilize by filter.</li>
			<li>Tfb2, icecold: 10 mM MOPS, 75 mM calcium chloride, 10 mM rubidium chloride, 15% glycerol; pH 6,6 achieving by
				adding diluted NaOH/ Sterilize by filter. </li>
			<li>Eppendorf cups</li>
			<li>Spectrofotometer</li>
			<li>Shaker at 37 degrees Celsius</li>
			<li>Centrifuge + tubes</li>
		</ul>
		<h2>Day before making competent cells:</h2>
		Put a bacteria into 2 ml LB medium at the beginning of the day. Shake at 125 rpm at 37 degrees celsius for at least 6
		hours.
		Dilute the culture 1:500 into 20 ml LB medium. (40 ul/20 ml). Shake at 125 rpm at 37 degrees celsius overnight.

		<h2>Day of making competent cells:</h2>
		<ol>
			<li>Dilute the overnight culture 1:100 into 200 ml LB medium (in a 1L flask). (2ml /200 ml)
			Shake the cells at 250 rpm and 37 degrees celsius till an OD600 of 0,5-0,7 is reached (this can take 2 hours).
			Measure every 30 minutes the OD600 with a cuvette and sterile LB medium as blanco.</li>

			<li>When a OD600 between 0,5-0,7 is reached. Cool the cells down to 4 degrees celsius and centrifuge the culture for 10
			minutes at 5000 x g and 4 degrees celsius. The cells need to stay at ice from this step on as much as possible. </li>

			<li>Dispose the supernatant and ad 66,4 ml cold TBF1 buffer /200 ml culture. Resuspended the pellet in the buffer
			completely.
			Centrifuge the culture for 10 minutes at 5000 x g and 4 degrees celsius.</li>

			<li>Dispose the supernatant and ad 24 ml cold TBF1 buffer /200 ml culture. Resuspended the pellet in the buffer
			completely. Incubate the cells on ice for 15 minutes.</li>

			<li>Centrifuge the culture for 10 minutes at 5000 x g and 4 degrees celsius.</li>

			<li>Dispose the supernatant and ad 4 ml cold TBF2 buffer /200 ml culture. Resuspended the pellet in the buffer
			completely. Incubate the cells on ice for 20 minutes.</li>

			<li>	Divide the solution over eppendorf cups by pipetting 50 ul per cup. Code the cups and place the cups on dry ice for
			10 minutes. After fully frozen, store the cells at -80 degrees celsius.</li>
		</ol>
	</div>

	<button class="accordion">Protocol 7 : Rapid DNA Dephos & Ligation </button>
	<div class="panel">
		<img src="http://2018.igem.org/wiki/images/1/1b/T--Rotterdam_HR--Protocol7.png" style="width: 450px; height: 700px; margin: auto; display: block;"> 
	</div>

	<button class="accordion">Protocol 8 : Digestion</button>
	<div class="panel">
		<ol>
			<li>For a double restriction enzyme digestion the following protocol was followed:</li>
			<li>Per 50 µL reaction mix the following:</li>
			<ul>
				<li>DNA: Up to 1 µg </li>
				<li>Cutsmart Buffer(10x): 5 µL</li>
				<li>Restriction enzyme one: 10 units</li>
				<li>Restriction enzyme two: 10 units</li>
			</ul>
			<li>Fill reaction to 50 µL with nuclease free water.</li>
			<li>Incubate at 37 degrees Celsius for two hours or overnight, depending on which restriction enzymes you use.</li>
			<li>Inactivate enzymes by incubating for 20 minutes at 65 degrees Celsius or by freezing the digestion solution for at least an hour by -20 degrees celsius.</li>
		</ol>
	</div>
	<button class="accordion">Protocol 9 : Glycerol stock</button>
	<div class="panel">
		<ul>
			<li>To make glycerol stocks a overnight culture was needed. The culture was made by putting a colony in LB-medium and incubate the culture for a night at 37ºC.</li>
			<li>After a night, the cultures are full enough and ready. To make a glycerol stock, 1 ml culture is mixed with 0,250 ml 87% glycerol solution. This is mixed by flicking the tube 3-5 times. The glycerol stock needs to be stored at -80ºC.</li>  
			<li>When a glycerol stock is used, be sure that the stock is not fully thawed before putting it back at -80ºC.</li>  
		</ul>
	</div>
	<button class="accordion">Protocol 10 : Mediums</button>
	<div class="panel">
		<h2>LB medium</h2>
		<ol>
			<li>Mix 5 grams of Lithium Broth combination with 200 ml demi water. After full dissolving, the medium needed to be sterilised by heating it to 121 degree celsius for at least 30 minutes. </li>
			<li>Store at 4 degrees Celsius :</li>
		</ol>
		<h2>LB agar</h2>
		<ol>
			<li>Per 200 ml LB medium, 3 gram Agar-Agar was added and dissolved. After full dissolving, the medium needed to be sterilised by heating it to 121 degree celsius for at least 30 minutes. </li>
			<li>Store at 4 degrees Celsius :</li>
		</ol>
		<h2>SOC medium</h2>
		<ol>
			<li>The following protocol is for 1L SOC- Medium </li>
			<li>Mix the following substances: </li>
			<ul>
					<li>2% (m/v) tryptone </li>
					<li>0,5% (m/v) yeast extract</li>
					<li>10mM  NaCl</li>
					<li>2,5 mM KCl</li>
			</ul>
			<li>Ad 980 ml demi water to this mixture. After full dissolving, the medium needed to be sterilised by heating it to 121 degree celsius for at least 30 minutes. </li>
			<li>Make the following solutions: </li>
			<ul>
				<li>10 mM MgCl2 (10 ml) </li>
				<li>20 mM Glucose (10 ml) </li>
			</ul>
				<li>Sterilize these solutions by filtering through a 0,22 um filter.</li>
				<li>Add both solutions aseptic to the premade medium. Be sure the medium was cooled down before adding the MgCl2 and Glucose. </li>
				<li>Store at 4 degrees celsius.</li>
		</ol>
		<h2>TAE buffer</h2>
		<ol>
		<li>This protocol is for a 50x TAE buffer.</li>
		<li>For use, we make a 1X solution by diluting it 50 times.</li>
		<ul>
			<li>Weigh out 242 grams of Tris-base (MW = 121.14 g/mol) and dissolve in approximately 700 milliliters of demi water. </li>
			<li>Carefully add 57.1 milliliters of 100 % glacial acid (or acetic acid) and 100 milliliters of 0.5 M EDTA (pH 8.0). </li>
			<li>Adjust the solution to a final volume of 1 liter.</li>
		</ul>
		<li>The pH of this buffer is not adjusted and should be about 8.5.</li>
		<li>Store stock solution at room temperature.</li>
	</ol>
	</div>
	<button class="accordion">Protocol 11 : PCR</button>
	<div class="panel">
		<ol>
			<li>Materials: For our PCR (Both plasmid and colony) reactions we have used the standard neb protocols for a 25 μl reaction , with the following concentrations:</li>
			<table>
				<tbody>
					<tr><td>10X Standard Taq Reaction Buffer</td><td>2.5 μl</td></tr>
					<tr><td>10 mM dNTPs</td><td>0.5 μl</td></tr>
					<tr><td>10 μM forward Primer (VF2)</td><td>0.5 μl</td></tr>                 	
					<tr><td>10 μM reverse Primers (VR) </td><td>0.5 μl</td></tr><tr>              	 
					<tr><td>Taq DNA Polymerase</td><td>	0.125 ul</td><tr>             	  
					<tr><td>Nuclease-free water</td><td>to 25 μl</td><tr>                    	 
				</tbody>
			</table>
			<li>Methods: A master mix is made with the concentrations shown above. In case of a plasmid pcr, a volume of plasmid is added before the reaction is filled with Nuclease-free water. With the colony PCR we used a toothpick to touch on the desired colony. Before briefly submerging and shaking the tip of the toothpick in the PCR Eppendorfcup filled with the mastermix. Before pricking it into a clean Agar plate a couple of times to grow more colonies in case the PCR shows us the desired products</li>
		</ol>
	</div>
</div>